src/components/Sidebar.tsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
-import "../assets/css/Sidebar.css";
+import "../assets/css/ProfilePage.css";
+import ProfilePage from "../pages/ProfilePage";
```

Comment:
```suggestion
import "../assets/css/Sidebar.css"
```
Apply sidebar styling to sidebar and remove unused imports